frontend/src/components/static_pages/about.jsx

```diff
@@ -1,24 +1,143 @@
-import React, { Component } from "react";
-import Markdown from "react-markdown";
-import Text from "./about.md";
+import React, { Component } from 'react';
+// import Markdown from 'react-markdown';
+// import Text from './Markdowns/about.md';
```

Comment:
lets just delete these lines instead of commenting them out -> we will always have the history in git 😄

Reply:
understand, everything should be cleaned up now.